tableauserverclient/server/request_options.py

```diff
@@ -146,10 +152,11 @@ class Orientation:
         Portrait = "portrait"
         Landscape = "landscape"
 
-    def __init__(self, page_type=None, orientation=None):
+    def __init__(self, page_type=None, orientation=None, maxage=None):
```

Comment:
Can we call it `max_age` here as well since it’s what you call it on the object anyway

Reply:
ImageRequestOptions already had maxage, so I wanted to keep it consistent across the 3 request objects without introducing any breaking changes. I guess we could deprecate it if we really wanted to change it?